src/com/twitter/intellij/pants/bsp/FastpassUtils.java

```diff
@@ -248,4 +207,32 @@ private static String toString(InputStream process) throws IOException {
         ))
     );
   }
+
+  /**
+   * Instead of of JDK9's CompletableFuture::onExit
+   */
+  @NotNull
+  public static CompletableFuture<Process> onExit(@NotNull Process process) {
+    return CompletableFuture.supplyAsync(() -> {
+      try {
+        Thread.sleep(15000);
```

Comment:
Feel free to ignore this suggestion
```suggestion
        Thread.sleep(java.util.concurrent.TimeUnit.SECONDS.toMillis(15));
```

Reply:
Can you explain why we sleep 15 seconds? What happens if the amend process completes faster than 15 seconds?

Reply:
Oh, sorry, I used it only for debugging! It should be removed